spacy/lang/nn/tokenizer_exceptions.py

```diff
@@ -0,0 +1,227 @@
+from ...symbols import NORM, ORTH
+from ...util import update_exc
+from ..tokenizer_exceptions import BASE_EXCEPTIONS
+
+_exc = {}
+
+
+for exc_data in [
+    {ORTH: "jan.", NORM: "januar"},
+    {ORTH: "feb.", NORM: "februar"},
+    {ORTH: "mar.", NORM: "mars"},
+    {ORTH: "apr.", NORM: "april"},
+    {ORTH: "jun.", NORM: "juni"},
+    {ORTH: "aug.", NORM: "august"},
+    {ORTH: "sep.", NORM: "september"},
+    {ORTH: "okt.", NORM: "oktober"},
+    {ORTH: "nov.", NORM: "november"},
+    {ORTH: "des.", NORM: "desember"},
+]:
+    _exc[exc_data[ORTH]] = [exc_data]
+
+
+for orth in [
+    "Ap.",
+    "Aq.",
+    "Ca.",
+    "Chr.",
+    "Co.",
+    "Dr.",
+    "F.eks.",
+    "Fr.p.",
+    "Frp.",
+    "Grl.",
+    "Kr.",
+    "Kr.F.",
+    "Kr.F.s",
+    "Mr.",
+    "Mrs.",
+    "Pb.",
+    "Pr.",
+    "Sp.",
+    "St.",
+    "a.m.",
+    "ad.",
+    "adm.dir.",
+    "adr.",
+    "b.c.",
+    "bl.a.",
+    "bla.",
+    "bm.",
+    "bnr.",
+    "bto.",
+    "c.c.",
+    "ca.",
+    "cand.mag.",
+    "co.",
+    "d.d.",
+    "d.m.",
+    "d.y.",
+    "dept.",
+    "dr.",
+    "dr.med.",
+    "dr.philos.",
+    "dr.psychol.",
+    "dss.",
+    "dvs.",
+    "e.Kr.",
+    "e.l.",
+    "eg.",
+    "eig.",
+    "ekskl.",
+    "el.",
+    "et.",
+    "etc.",
+    "etg.",
+    "ev.",
+    "evt.",
+    "f.",
+    "f.Kr.",
+    "f.eks.",
+    "f.o.m.",
+    "fhv.",
+    "fk.",
+    "foreg.",
+    "fork.",
+    "fv.",
+    "fvt.",
+    "g.",
+    "gl.",
+    "gno.",
+    "gnr.",
+    "grl.",
+    "gt.",
+    "h.r.adv.",
+    "hhv.",
+    "hoh.",
+    "hr.",
+    "ifb.",
+    "ifm.",
+    "iht.",
+    "inkl.",
+    "istf.",
+    "jf.",
+    "jr.",
+    "jun.",
```

Comment:
So you still have the abbreviation `jul.`, just not with the `NORM` `juli`?

Since `jun.` is above, I think this list can just be modified to have `jul.` instead:

```suggestion
    "jul.",
```

Reply:
I'm not sure whether you ended to leave "jul." out as an abbreviation entirely. Since `nb` does include it, I decided to make the same decision to keep it here, just without the `NORM`, even though "jul." at the end of a sentence would be tokenized as intended for the non-July/abbreviation usage.